doc/neps/nep-0056-array-api-main-namespace.rst

```diff
@@ -272,15 +288,31 @@ same name. We may deprecate ``np.trace`` and ``np.diagonal`` to resolve it, but
 preferably not immediately to avoid users having to write ``if-2.0-else``
 conditional code.
 
-There may be a few other changes that don't quite fall in one of the categories
-above. For example, ``numpy.fft`` functions need to preserve precision for
-32-bit input dtypes rather than upcast to ``float64``/``complex128`` (desirable
-anyway, and can be supported with the new gufunc implementation in 
-(`gh-25336 <https://github.com/numpy/numpy/pull/25336>`__) . Also in
-``numpy.fft``, there's an issue with the ``s``/``axes`` argument in n-D
-transforms that needs solving
-(see `gh-25495 <https://github.com/numpy/numpy/pull/25495>`__).
+*We expect the impact of this category of changes to be small.*
+
+**The semantics of the copy keyword in asarray and array** for
+``copy=False`` will change from "copy if needed" to "never copy". there are now
+three types of behavior rather than two - ``copy=None`` means "copy if needed".
 
+*We expect the impact of this category of changes to be medium. In case users get
+an exception because they use* ``copy=False`` *explicitly in their copy but a
+copy was previously made anyway, they have to inspect their code and determine
+whether the intent of the code was the old or the new semantics (both seem
+rougly equally likely), and adapt the code as appropriate. We expect most cases
+to be* ``np.array(..., copy=False)``, *because until a few years ago that had
+lower overhead than* ``np.asarray(...)``. *This was solved though, and*
+``np.asarray(...)`` *is idiomatic NumPy usage.*
+
+**Other changes**: there may be a few other changes that don't quite fall in
+one of the categories above. For example, ``numpy.fft`` functions need to
+preserve precision for 32-bit input dtypes rather than upcast to
+``float64``/``complex128`` (desirable anyway, and can be supported with the new
+gufunc implementation in (`gh-25336
+<https://github.com/numpy/numpy/pull/25336>`__) . Also in ``numpy.fft``,
+there's an issue with the ``s``/``axes`` argument in n-D transforms that needs
+solving (see `gh-25495 <https://github.com/numpy/numpy/pull/25495>`__).
```

Comment:
gh-25495 has also been merged.

Reply:
Yes, great that these have been merged. I'm not sure that should change the text here though? Ideally it reads okay whether or not linked PRs have been merged or not.

Reply:
It just seemed the text would read more easily if adjusted. (and the first link points to an unrelated PR...)

Reply:
Okay, you are right. I rewrote this section now and make it explicitly about `numpy.fft` rather than putting it under "other changes".